docs/docs/quick-start.md

```diff
@@ -4,63 +4,40 @@ title: Quick Start
 
 This quick start is intended for intermediate to advanced developers. For a gentler intro to Gatsby, [head to our tutorial](/tutorial/)!
 
-## Use the Gatsby CLI
+## Getting started with Gatsby
 
-<EggheadEmbed
-  lessonLink="https://egghead.io/lessons/gatsby-quick-start-with-gatsby-from-the-command-line-5bf2403a"
-  lessonTitle="Quick Start with Gatsby: Create, Develop, and Build Gatsby Sites From the Command Line"
-/>
-
-### Install the Gatsby CLI
+1. Create a new site
 
 ```shell
-npm install -g gatsby-cli
+npm init gatsby
 ```
 
-> The above command installs Gatsby CLI globally on your machine.
-
-### Create a new site
-
-```shell
-gatsby new gatsby-site https://github.com/gatsbyjs/gatsby-starter-hello-world
-```
+Follow the prompts to choose your preferred CMS, styling tools and additional features.
 
-### Change directories into site folder
+2. Change directories into site folder
 
 ```shell
 cd gatsby-site
```

Comment:
Does `npm init gatsby` ask for the project directory? It might be good to add some text to this step to say where "gatsby-site" comes from.

Reply:
It does, but the prompt is: What would you like to name the folder where your site will be created?

That feels like it provides good context to me? But open to thoughts.

Reply:
Made a suggestion a couple lines up. Maybe something like that?

Reply:
Perfect!